http/events_test.go

```diff
@@ -135,147 +135,6 @@ func TestEventsSubscribe(t *testing.T) {
 	}
 }
 
-// TestEventsSubscribeNamespaces tests the websocket endpoint for subscribing to events in multiple namespaces.
-func TestEventsSubscribeNamespaces(t *testing.T) {
```

Comment:
I can see why this test was moved to the enterprise repo given the namespaces, but maybe worth keeping a subset of it that only tests the root namespace in this repo to catch regressions early?